supersetapiclient/base.py

```diff
@@ -45,7 +45,6 @@ def raise_for_status(response):
 
 class Object:
     _parent = None
-    EXPORTABLE = False
```

Comment:
Why should we remove this ?

Reply:
Because all currently supported entities are exportable in Superset 2.0

Reply:
I haven't added support for testing other Superset versions than 2.0. We could do that, but Superset is moving pretty fast and I'm not sure it's worth the effort making the package backwards-compatible.